README.md

```diff
@@ -76,6 +94,34 @@ settings.json:
 }
 ```
 
+## Npm Projects
+
+The extension builds the npm dependencies tree by running `npm list`. View licenses and top issue severities directly from package.json.
+To scan your npm dependencies, make sure to fulfill the following requirements:
+
+1. npm cli is installed in your system PATH.
+1. Your project is installed by running `npm install`.
+
+## Go Projects
+
+The extension build the Go dependencies tree by running `go mod graph`. View licenses and top issue severities directly from go.mod.
+To scan your Go dependencies, just make sure to have Go CLI in your system PATH.
+
+## Pypi Projects
+
+The extension build the Pypi dependencies tree by running `pipdeptree` on your Python virtual environment. It also takes the Python interpreter path from the [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python). View licenses and top issue severities directly from your requirements.txt files. The scan your Pypi dependencies, make sure to fulfill the following requirements:
```

Comment:
It also takes the Python interpreter path from the Python extension --> It also uses the Python interpreter path configured by the [Python extension]